Content.Shared/Prayer/PrayableComponent.cs

```diff
@@ -30,6 +31,34 @@ public sealed partial class PrayableComponent : Component
     [ViewVariables(VVAccess.ReadWrite)]
     public string NotificationPrefix = "prayer-chat-notify-pray";
 
+    /// <summary>
+    /// The sound played for admins when a prayer is sent
+	/// </summary>
+    [DataField("soundForGod")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public SoundSpecifier SoundForGod = new SoundPathSpecifier("/Audio/Effects/plehnimda.ogg");
```

Comment:
all datafields are automatically rw now
and tag isnt needed

so just `[DataField]`